deploy/helm/sumologic/templates/_helpers/_common.tpl

```diff
@@ -311,6 +311,7 @@ If dryRun=true, we do not create any chart labels.
 chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
 release: "{{ .Release.Name }}"
 heritage: "{{ .Release.Service }}"
+app.kubernetes.io/instance: "{{ .Release.Name }}"
```

Comment:
This was added because kube-state-metrics and node-exporter charts don't add the `release` label to their ServiceMonitors, and I needed a label to select for. In the near future, this shouldn't be necessary, as otel operator will expose more powerful selectors for this.